aggregator/aggregator.go

```diff
@@ -185,6 +207,15 @@ func aggregate(context *cli.Context) error {
 					errMsg := fmt.Sprintf("Node %s: %s is %s\n", node.Address, curr.Type, curr.Result)
 					log.Warning(errMsg)
 					nodeHealthy = false
+
+					if _, ok := blacklistMap[curr.Type]; ok {
```

Comment:
This whole block is kind of confusing.  Can we add a nice meaty comment here explaining that it's using inverse logic, if that's the case?